.github/workflows/ci.yml

```diff
@@ -3,6 +3,12 @@ name: CI
 on: [push, pull_request]
 
 jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+
```

Comment:
We need the `markdown-link-check` to happen _after_ the `build` step, because we need to test it on the newly generated files?

See: https://github.com/ARM-software/acle/pull/43/checks